### PR TITLE
Create class diagram for project

### PR DESCRIPTION
**Pull request by Giang-github**
Created class diagram for project

### CODE REVIEW

File: src/main/java/org/resitrack/entity/House.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.resitrack.entity;
+
+import java.util.List;
+
+public class House {
+    private List<Person> members;
+    private int numberHouse;
+    private String householdHead;
```
Comment (ntananh) [inline src/main/java/org/resitrack/entity/House.java:8]: I think `householdId `is a better name for variable holding a household id value. What do you think?
```suggestion
    private String householdId;
```

File: src/main/java/org/resitrack/entity/Person.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import org.resitrack.enums.Gender;
+
+import java.time.LocalDate;
+
+public class Person {
+    private String name;
+    private Gender sex;
+    private LocalDate dob;
+    private long idNumber;
```
Comment (ntananh) [inline src/main/java/org/resitrack/entity/Person.java:11]:
```suggestion
    private String id;
```
the id should a String, we will discuss about the id generation for each person with some random string

Comment (ntananh) [inline src/main/java/org/resitrack/entity/Person.java:11]: Could you please re-arrage the properties location

File: src/main/java/org/resitrack/entity/Town.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package org.resitrack.entity;
+
+import java.util.List;
+
+public class Town {
+    private List<House> houseList;
+    private int zipCode;
+    private String name;
```
Comment (ntananh) [inline src/main/java/org/resitrack/entity/Town.java:8]:
```suggestion
    private String name;
    private int zipCode;
    private List<House> houseList;
    
```

File: src/main/java/org/resitrack/entity/Town.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.resitrack.entity;
+
+import java.util.List;
+
+public class Town {
```
Comment (ntananh) [inline src/main/java/org/resitrack/entity/Town.java:5]: Town should have Id propertity

File: src/main/java/org/resitrack/enums/Gender.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package org.resitrack.enums;
+
+public enum Gender {
```
Comment (ntananh) [inline src/main/java/org/resitrack/enums/Gender.java:3]: Could you please add the java doc tell what is this enum used for

File: src/main/java/org/resitrack/entity/House.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+package org.resitrack.entity;
+
+import java.util.List;
+
+public class House {
```
Comment (ntananh) [inline src/main/java/org/resitrack/entity/House.java:5]: I think House should has more properties like (id, address,....). Could you add more propertites, and rearrage the place of variables

File: src/main/java/org/resitrack/entity/House.java [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+    private List<Person> members;
+    private int numberHouse;
```
Comment (ntananh) [inline src/main/java/org/resitrack/entity/House.java:7]: The `numberHouse` in the real case i think we should use `String` type to store becaue in the real life house number may be contains the characters on it (eg: 48A, 25C)

**Comment by ntananh** [comment]
I have finished my first reivew.

File: src/main/java/org/resitrack/entity/Person.java (hunk 2 above)
Comment (Giang-github) [reply src/main/java/org/resitrack/entity/Person.java:11]: idNumber is an attribute that represents a person's identity card number
I will add id attribute with data type as String

File: src/main/java/org/resitrack/entity/Town.java (hunk 4 above)
Comment (Giang-github) [reply src/main/java/org/resitrack/entity/Town.java:5]: zipCode is the city's code as the only attribute. We can determine city name and other attributes via zipCode without id

**Pull request closed without merging**